tests/python/frontend/pytorch/test_forward.py

```diff
@@ -4055,6 +4155,8 @@ def forward(self, x):
     test_hard_sigmoid()
     test_forward_nll_loss()
     test_forward_flip()
+    test_forward_grid_sampler()
+    test_forward_float()
```

Comment:
forgot to add test_forward_im2col.

Reply:
Thanks @wyc-ruiker, now we add it.

Reply:
we also need to add `verify_script_model` in https://github.com/apache/tvm/pull/8443#discussion_r667597493

Reply:
@wyc-ruiker , 1)sorry for commit lost, now add Im2col3x3 test, this is for trace mode, Im2col5x5 for script mode;
2)we use separate modules just for cover trace/script mode and keep simple as possible.
Thanks a gain.